implementation/src/main/java/io/smallrye/metrics/setup/config/HistogramBucketMinConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package io.smallrye.metrics.setup.config;
+
+import java.util.ArrayDeque;
+import java.util.Collection;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class HistogramBucketMinConfiguration extends PropertySingleValueConfiguration<Double> {
+
+    private static final String CLASS_NAME = HistogramBucketConfiguration.class.getName();
+    private static final Logger LOGGER = Logger.getLogger(CLASS_NAME);
+
+    public HistogramBucketMinConfiguration(String metricName, Double value) {
+        super(metricName, value);
+    }
+
+    public static Collection<HistogramBucketMinConfiguration> parse(String input) {
+
+        ArrayDeque<HistogramBucketMinConfiguration> metricBucketMinMax = new ArrayDeque<HistogramBucketMinConfiguration>();
+
+        if (input == null || input.length() == 0) {
+            return null;
+        }
+
+        // not expecting backslashes?
+        String[] metricValuePairs = input.split(";");
+
+        // Individual metric name grouping and values
+        for (String kvString : metricValuePairs) {
+
+            String[] keyValueSplit = kvString.split("=");
+
+            String metricName = keyValueSplit[0];
+
+            HistogramBucketMinConfiguration metricBucketConfiguration = null;
+
+            // metricGroup=<blank> == invalid
+            if (keyValueSplit.length == 2) {
+                if (keyValueSplit[1].matches(("[0-9]+[.]*[0-9]*"))) {
```

Comment:
Does the dot need escape?

Reply:
the `[  ]` character set allows for matching explicitly the characters within it.